### PR TITLE
Adapt for wAR

### PR DESCRIPTION
**Pull request by ujnss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,9 +287,10 @@ Submit a task to AO and get the result. The combination of submitTask and getRes
 
 ### getComputationPrice
 
-Get the computing price of each node for each task. Now only supports AO’s test token AOCRED, minimum unit to use AOCRED(1 means 0.001 AOCRED).
+Get the computing price of each node for each task. Now only supports wAR(the Wrapped AR in AO), minimum unit to use wAR(1 means 0.000000000001 wAR).
 
-* **Parameters:** NULL.
+* **Parameters:** 
+  * `symbol:number(optional)` The price symbol (default: wAR).
```
Comment (fksyuan) [inline README.md]: symbol:number should be symbol:string

File: src/index.ts [hunk 2]
```diff
@@ -172,20 +172,27 @@ export const submitTask = async (dataId: string, dataUserPk: string, wallet: any
   const nodeNames = exData.policy.names;
   const priceObj = JSON.parse(encData.price);
   const symbol = priceObj.symbol;
-  if (symbol !== 'AOCRED') {
-    throw new Error('Only support AOCRED now');
+  if ((symbol === 'AOCRED') || (symbol === 'wAR')) {
```
Comment (fksyuan) [inline src/index.ts]: The if statement can change to if ((symbol != 'AOCRED') && (symbol != 'wAR')) throw

**Review by fksyuan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ujnss) [reply README.md]: Fixed. See [363fbcb](https://github.com/pado-labs/pado-ao-sdk/pull/57/commits/363fbcbd5d85fb9c24f4e910e339a9cb18cdd95c).

File: src/index.ts (hunk 2 above)
Comment (ujnss) [reply src/index.ts]: Refined.

**Review by fksyuan — APPROVED** [review]

**Pull request merged**